lib/circuit-breaker.ts

```diff
@@ -0,0 +1,233 @@
+// Based on https://github.com/yammer/circuit-breaker-js
```

Comment:
Please update the License with assigned copyright to yammer.

Something like this would work.
https://github.com/zalando-incubator/graphql-jit/blob/master/LICENSE

Reply:
Good point, added.